src/main/java/org/folio/de/entity/bursarlegacy/LegacyJob.java

```diff
@@ -0,0 +1,102 @@
+package org.folio.de.entity.bursarlegacy;
```

Comment:
Should this have landed in git? Or is it generated?

Reply:
Yeahh, it's how basically how JPA maps the database rows to Java objects. There's [a similar one](https://github.com/ualibweb/mod-data-export-spring/blob/master/src/main/java/org/folio/de/entity/Job.java) for the new `Job` as well.

Reply:
Ah right; just wanted to make sure we weren't duplicating generated code! LGTM